independent-publisher-connectors/Spotify/apiDefinition.swagger.json

```diff
@@ -0,0 +1,943 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Spotify",
+    "description": "The Spotify connector will give you access to your Library, Artists, Albums, Tracks and more!",
+    "version": "1.0",
+    "contact": {
+      "name": "Daniel Laskewitz | Sogeti",
+      "url": "https://youtube.com/daniellaskewitz",
+      "email": "connectorsupport@laskewitz.io"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://spotify.com"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.spotify.com/us/legal/privacy-policy/"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Lifestyle and Entertainment;Social Media"
+    }
+  ],
+  "host": "api.spotify.com",
+  "basePath": "/",
+  "schemes": ["https"],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v1/me/albums": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "items": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/definitions/Album"
+                  },
+                  "x-ms-summary": "Albums",
+                  "description": "Get a list of the albums saved in the current Spotify user's 'Your Music' library."
+                },
+                "href": {
+                  "type": "string",
+                  "x-ms-summary": "Href",
+                  "description": "A link to the Web API endpoint returning the full result of the request"
+                },
+                "limit": {
+                  "type": "integer",
+                  "format": "int32",
+                  "x-ms-summary": "Limit",
+                  "description": "The maximum number of items in the response (as set in the query or by default)."
+                },
+                "next": {
+                  "type": "string",
+                  "x-ms-summary": "Next",
+                  "description": "URL to the next page of items. ( null if none)"
```

Comment:
@Laskewitz Can you please remove the extra space in the rounded brackets.

`"description": "URL to the next page of items. (null if none)"`

Reply:
Fixed.